packages/core/integration-tests/test/cache.js

```diff
@@ -4374,85 +4374,217 @@ describe('cache', function () {
               );
             },
             async update(b) {
-              let html = await overlayFS.readFile(
-                b.bundleGraph.getBundles().find(b => b.name === 'b.html')
-                  ?.filePath,
-                'utf8',
-              );
-              assert.equal(html.match(/<script/g)?.length, 7);
-
+              assertBundles(b.bundleGraph, [
```

Comment:
We changed the cache tests because they didn't seem to test the config properly, now the test for `adding a config` and `removing` determine if minBundleSize was upheld, for bundles with a large asset.
 
The previous test was testing script tags within a bundle, so it didn't seem like it was **just** testing  bundler config.